internal/testcafe/config.go

```diff
@@ -0,0 +1,97 @@
+package testcafe
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/rs/zerolog/log"
+	"github.com/saucelabs/saucectl/internal/config"
+	"gopkg.in/yaml.v2"
+)
+
+// Project represents the testcafe project configuration.
+type Project struct {
+	config.TypeDef `yaml:",inline"`
+	ShowConsoleLog bool
+	Sauce          config.SauceConfig `yaml:"sauce,omitempty" json:"sauce"`
+	Suites         []Suite            `yaml:"suites,omitempty" json:"suites"`
+	BeforeExec     []string           `yaml:"beforeExec,omitempty" json:"beforeExec"`
+	Docker         config.Docker      `yaml:"docker,omitempty" json:"docker"`
+	Testcafe       Testcafe           `yaml:"testcafe,omitempty" json:"testcafe"`
+	Npm            config.Npm         `yaml:"npm,omitempty" json:"npm"`
+}
+
+// Suite represents the testcafe test suite configuration.
+type Suite struct {
+	Name             string            `yaml:"name,omitempty" json:"name"`
+	Browser          string            `yaml:"browser,omitempty" json:"browser"`
```

Comment:
I'm in favor of `browserName` here, since `browser` is already not the native testcafe attribute, we might as well choose the the name that's common to the attribute we use in other configs 💡

Reply:
make sense!